### ISSUE TITLE
Mountain Worm Clones

### ISSUE THREAD

**Issue by HunnyDoo** [EdenServer/community#1900]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details

ver30181205_0

https://ffxiclopedia.fandom.com/wiki/Mountain_Worm

The 21-24 hour nm Mountain Worm shares its name with a regular enemy also called Mountain Worm. The NM version is charcoal colored instead of brown and drops a set of gloves called Heavy Gauntlets. Currently every single brown Mountain Worm in Uleguerand checks Impossible to Gauge, drops the gloves, drops gil, and respawns in 5 minutes.

**Comment by danismyhero** [comment]
This now occurs with the Kindred tier of Demons as well.  I believe this happened when Dynamis - Xarcabard was added (they share a name with those mobs), but I had never checked the mobs, only noticed they stayed over my body when I died.  I do not believe they share a loot table, as they provide exp, gil, and I was able to get a Demon Horn from a Kindred Black mage.

They do seem to have the behavior of some NM or dynamis mobs, such as not despawning, or not moving once killing you.  This includes outside their normal roaming range making it impossible to reraise as they agro by sight,

None used 2-hours like in Dynamis.
